website/content/en/status/report-2023-04-2023-06/gcc.adoc

```diff
@@ -19,12 +19,14 @@ With the release of GCC 13.1 (first GCC 13 release: I remind that GCC counts min
 
 ==== The *-devel ports
 
-A few significant changes have been made on the *-devel ports, which will be forwarded to the production ports together with more changes (see below):
+Enabled support for .init_array and .fini_array, which FreeBSD supports since commit gitref:83aa9cc00c2d83d05a0efe7a1496d8aab4a153bb[repository=src].
 
-* enable support for .init_array and .fini_array, which FreeBSD supports since commit gitref:83aa9cc00c2d83d05a0efe7a1496d8aab4a153bb[repository=src];
+Reverted from LTO_BOOTSTRAP to STANDARD_BOOTSTRAP as the default option on i386, amd64 and aarch64: 
 
-* switch back to STANDARD_BOOTSTRAP on i386, amd64 and aarch64 as default option instead of LTO_BOOTSTRAP: LTO bootstrap produces too many failures on the package builders for those architectures.
-Users wanting to use LTO_BOOTSTRAP can still enable the option manually.
+- LTO bootstrap produces too many failures on the package builders for those architectures
+- LTO_BOOTSTRAP remains available for users who want it.
+
+Those changes and others will be forwarded to the production ports.
```

Comment:
Which "others"? You might be right, but I do not remember others. I would go with "Those changes will..."

Reply:
I probably assumed others through the original mention of "A few".

[`6d5c37c` (#206)](https://github.com/freebsd/freebsd-doc/pull/206/commits/6d5c37c45ba32d97a4345ba8e4140718fdf85b70) hopefully OK.